### PR TITLE
Prepare IOS 8.0.3 and CI changes for IOS

### PR DESCRIPTION
**Pull request by jigar-f**
This PR updates, IOS CI changes, and fixes issues with tags.

### CODE REVIEW

**Comment by jigar-f** [comment]
This is yet to be tested on the tag, I will test this tomorrow.

**Comment by jigar-f** [comment]
@atavism Can you review this? I like this to get merged.

**Review by atavism — APPROVED** [review]

**Comment by atavism** [comment]
LGTM if you just want to fix the merge conflict

File: Makefile [hunk 1]
```diff
@@ -449,9 +449,9 @@ set-version:
 	NEXT_BUILD=$$(($$CURRENT_BUILD + 1)); \
 	/usr/libexec/PlistBuddy -c "Set :CFBundleVersion $$NEXT_BUILD" $(INFO_PLIST)
 
-ios: macos build-framework ffigen
+#ios: macos build-framework ffigen
```
Comment (atavism) [inline Makefile:452]: Is this supposed to be commented out? There's a step in CI that depends on this to build the Lantern library:

```
      - name: Build Lantern Library
        shell: bash
        run: make ${{matrix.platform}}
        env:
          VERSION: ${{ env.version }}
```

We still need to do `make build-framework`, right?

Comment (jigar-f) [reply Makefile:452]: No we don't, build-framework renamed to ios.

**Pull request merged**